Lib/concurrent/futures/process.py

```diff
@@ -855,3 +868,65 @@ def shutdown(self, wait=True, *, cancel_futures=False):
         self._executor_manager_thread_wakeup = None
 
     shutdown.__doc__ = _base.Executor.shutdown.__doc__
+
+    def _terminate_or_kill_workers(self, operation: _TerminateOrKillOperation):
+        """Attempts to terminate or kill the executor's workers based off the given
+        operation. Iterates through all of the current processes and performs the
+        relevant task if the process is still alive.
+
+        After terminating workers, the pool will be in a broken state
+        and no longer usable (for instance, new tasks should not be
+        submitted).
+        """
+        if operation not in _TerminateOrKillOperation._member_map_.values():
```

Comment:
```suggestion
        if operation not in _TerminateOrKillOperation:
```

I rhink it's sufficient but please check.